src/popup/sass/components/_popup.scss

```diff
@@ -141,3 +141,6 @@
   background: #cb2027;
   color: white;
 }
+.removeLoadMore{
```

Comment:
There is already a `display-none` rule defined in `_utilities.scss`. Please use that.

Reply:
This was indeed my initial approach, I was trying to use `display-none` for hiding the button, but I don't know why it wasn't working. After hours of debugging I couldn't find the bug so I had to create `removeLoadMore` which gave me the desired result.

Reply:
I tried again but couldn't do it. I can't really tell what's causing this behaviour , should I continue using removeLoadMore ?

Reply:
Okay, we can improve upon that later. I have added a couple more comments, please look into it. We can merge this after that.